runtime/jcl/common/zaap.c

```diff
@@ -0,0 +1,56 @@
+/*******************************************************************************
+ * Copyright IBM Corp. and others 2023
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] https://openjdk.org/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0 OR GPL-2.0-only WITH OpenJDK-assembly-exception-1.0
+ *******************************************************************************/
+
+#include "j9.h"
+#include "jcl_internal.h"
+
+#if defined(J9VM_OPT_JAVA_OFFLOAD_SUPPORT)
+/**
+ * Clear the non-ZAAP eligible bit for JCL natives to allow them to run on zAAP.
+ *
+ * @param[in] env                 The JNI interface pointer
+ * @param[in] nativeClass         The class containing the native method
+ * @param[in] nativeMethods       The JNI native method pointer
+ * @param[in] nativeMethodCount  The count of native methods
+ */
+void
+clearNonZAAPEligibleBit(JNIEnv *env, jclass nativeClass, const JNINativeMethod *nativeMethods, jint nativeMethodCount)
+{
+	J9VMThread *vmThread = (J9VMThread *) env;
+	J9JavaVM *vm = vmThread->javaVM;
+	J9InternalVMFunctions* vmFuncs = vm->internalVMFunctions;
```

Comment:
Please tidy up:
* no space after a cast (like line 50)
* space before, but not after `*` in declarations